chroniques-des-activites-archeologiques-de-l-ecole-francaise-de-rome.csl

```diff
@@ -102,6 +132,26 @@
       </else>
     </choose>
   </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <text value="$$$"/>
```

Comment:
could you explain this? I'd rather not have a required search&replace in an official style. What is $$$ and $$$$ standing in for exactly?

Reply:
They want the indication of the first edition to be translated in the used language:
1<sup>re</sup> éd. in French
1<sups>a</sup> ed. in Italian
1<sup>st</sup> ed. in English
It does seem to render it automatically, does it?
Hence the specification in the info:
"En cas de réédition du volume, il faut remplacer les $$$ par la forme "1ère éd. [date]" manuellement, entre parenthèses, en fin de référence; la date de première édition doit être entrée dans le champ 'Extra' sous la forme 'Original Date: [date]' (voir https://www.zotero.org/support/kb/item_types_and_fields#citing_fields_from_extra)". I could translate it in English and Italian, but it should then be indicated on the review's site

Reply:
Answer edited... Words were lacking...

Reply:
Unless I'm misunderstanding, `<number variable="edition" form="ordinal"/>` should work just fine. 

But it seems like you're conflating two things here -- the date of the 1st edition and the current edition number, or am I misunderstanding something?

Reply:
In fact, the editor is not interested in the current edition number, cf. e.g. this book which was then at its second edition:
Style generated:
Étienne 1977 = R. Étienne, _La vie quotidienne à Pompéi_, Paris, 1977, (_La vie quotidienne_, $$$ éd. 1966).
Asked by the editor:
Étienne 1977 = R. Étienne, _La vie quotidienne à Pompéi_, Paris, 1977, (_La vie quotidienne_, 1<sup>re</sup> éd. 1966).

Reply:
In that case, you should presumably test for `original-date` first before displaying the edition. Otherwise this would just be 

> Étienne 1966 = R. Étienne, La vie quotidienne à Pompéi, Paris, 1966

With no mention of the edition at all.

Since that means it'll always say 1st edition (in whatever language) I would suggest just rendering the ordinal in French and having people swap it out for Italian and English if needed. I really dislike the required search & replace. You can't assume that people read the summary of the style or even details on the publisher page.

Reply:
Ok, but it will be hard-coded